### ISSUE TITLE
Transitive dependency Xerces has vulnerability CVE-2022-23437

### ISSUE THREAD

**Issue by sandeepnkulkarni** [stanfordnlp/CoreNLP#1264]
Xerces is a transitive dependency coming via xom into CoreNLP. It has a vulnerability CVE-2022-23437 and needs to be upgraded to 2.12.2.
I can see that xom has no new release which upgrades Xerces till this date.

**Comment by AngledLuffa** [comment]
Can you post an issue with xom?  We have no control over it, and there's
basically no chance we try to remove it

**Comment by sandeepnkulkarni** [comment]
Yes, I have done that as well: https://github.com/elharo/xom/issues/175

**Comment by AngledLuffa** [comment]
xerces isn't actually in our releases or anywhere in our pom files, right?  aside from replacing xom, we're completely incapable of doing anything to affect this.  if xom gets updated and we don't notice, feel free to ping us and we'll update

**Comment by VishalGokhale** [comment]
@AngledLuffa 
Xom has been updated to 1.3.8. 
and xalan has been made optional. 
xalan has [vuln CVE-2022-34169](https://www.mend.io/vulnerability-database/CVE-2022-34169)

**Comment by AngledLuffa** [comment]
Thanks, we can give that a try:

https://github.com/stanfordnlp/CoreNLP/commit/3ded6f03f3956e83439755351048401dae85bf72

**Comment by AngledLuffa** [comment]
@VishalGokhale Apparently maven is still importing xalan from xom.  Looking at the xom pom, I don't see any reason to believe xalan is optional:

https://repo1.maven.org/maven2/xom/xom/1.3.8/xom-1.3.8.pom
